### ISSUE TITLE
cmd/trayscale: more peer utilities and info

### ISSUE THREAD

**Issue by DeedleFake** [DeedleFake/trayscale#5]
Peers currently only show their names and a list of their Wireguard IP addresses. More features would be good. Here are a few:

* More information just pulled straight from `ipnstate.PeerStatus`, such as `ID`, `OS`, and `Active`.
* A better indicator that the top peer is the current machine.
* Ability to send and receive files from a peer, possibly? This one's potentially messy.

**Comment by vampywiz17** [comment]
good idea! it would be good to see that is direct connection to the peer or use darp

**Comment by Jacalz** [comment]
Just having a little circle that indicates grey or green, like in this example from the iOS app, would be a huge improvement.
![image](https://github.com/DeedleFake/trayscale/assets/25466657/72bfb6bf-a37f-447e-ad3a-094c589c92f5)

**Comment by Jacalz** [comment]
The new status indicators are a nice addition. There seems to be a too much padding on the left compared to the right. If we look at the image above, we can see that the padding to the left screen edge and the peer name is about the same. Maybe one or two more spaces to the right of the emoji would improve the look?

It would also be nice to have flat circles instead of the shiny emoji icons, with a gray circle instead of red to indicate a disconnected device. Red is a little more of an error colour in my opinion.

**Comment by DeedleFake** [comment]
> It would also be nice to have flat circles instead of the shiny emoji icons, with a gray circle instead of red to indicate a disconnected device. Red is a little more of an error colour in my opinion.

Unfortunately, until the peer list is replaced with a more formattable widget (#63), the emoji are all that's available.

I considered gray instead of red but went with red instead to make it stand out from the background more, but as I use it I kind of agree that red isn't great. Ironically, it kind of stands out too much. I'll experiment a bit more and see how well gray works.

**Comment by DeedleFake** [comment]
I tried a white circle  and yeah, like I was worried about, it really blends into the background pretty badly in light mode.

![image](https://github.com/DeedleFake/trayscale/assets/326750/701f9cd3-bdc7-4c51-9c2f-bf2a8fbed525)

Unfortunately, there doesn't actually seem to be a gray circle emoji. I think making this nicer is going to have to wait on #63.
